src/backend/replication/logical/logical.c

```diff
@@ -1236,6 +1236,8 @@ message_cb_wrapper(ReorderBuffer *cache, ReorderBufferTXN *txn,
 
 	if (ctx->callbacks.message_cb == NULL)
 		return;
+	if (strstr(prefix, "neon-file") != NULL)
```

Comment:
Why `strstr`?
`neon_file:` is prefix, so it is much better to use just `strncmp(prefix, "neon-file:", 9) == 0`

Reply:
Changed it